web/src/i18n/locales/fr.json

```
@@ -11,6 +11,11 @@
     " 吗？": " ?",
     " 秒": "s",
     " 秒。": " secondes.",
+    "，当前无生效订阅，将自动使用钱包": ", aucun abonnement actif, le portefeuille sera utilisé automatiquement.",
```

Biome (2.4.10)

[error] The key ，当前无生效订阅，将自动使用钱包 was already declared.

(lint/suspicious/noDuplicateObjectKeys)